[2024-04-28T16:08:06.167+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-04-28T16:08:06.183+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: Final_exam.ending manual__2024-04-28T16:06:30.964075+00:00 [queued]>
[2024-04-28T16:08:06.191+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: Final_exam.ending manual__2024-04-28T16:06:30.964075+00:00 [queued]>
[2024-04-28T16:08:06.191+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2024-04-28T16:08:06.203+0000] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): ending> on 2024-04-28 16:06:30.964075+00:00
[2024-04-28T16:08:06.211+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=134) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-04-28T16:08:06.213+0000] {standard_task_runner.py:63} INFO - Started process 136 to run task
[2024-04-28T16:08:06.213+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'Final_exam', 'ending', 'manual__2024-04-28T16:06:30.964075+00:00', '--job-id', '28', '--raw', '--subdir', 'DAGS_FOLDER/Project.py', '--cfg-path', '/tmp/tmpetzv_wq2']
[2024-04-28T16:08:06.214+0000] {standard_task_runner.py:91} INFO - Job 28: Subtask ending
[2024-04-28T16:08:06.250+0000] {task_command.py:426} INFO - Running <TaskInstance: Final_exam.ending manual__2024-04-28T16:06:30.964075+00:00 [running]> on host 73ada3aeb297
[2024-04-28T16:08:06.315+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='ha20040204@gmail.com' AIRFLOW_CTX_DAG_OWNER='HA' AIRFLOW_CTX_DAG_ID='Final_exam' AIRFLOW_CTX_TASK_ID='ending' AIRFLOW_CTX_EXECUTION_DATE='2024-04-28T16:06:30.964075+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-28T16:06:30.964075+00:00'
[2024-04-28T16:08:06.316+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-04-28T16:08:06.317+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-04-28T16:08:06.318+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "Completed"']
[2024-04-28T16:08:06.331+0000] {subprocess.py:86} INFO - Output:
[2024-04-28T16:08:06.332+0000] {subprocess.py:93} INFO - Completed
[2024-04-28T16:08:06.332+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-04-28T16:08:06.333+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-04-28T16:08:06.350+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=Final_exam, task_id=ending, execution_date=20240428T160630, start_date=20240428T160806, end_date=20240428T160806
[2024-04-28T16:08:06.387+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-04-28T16:08:06.399+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-04-28T16:08:06.401+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
